code/controllers/subsystem/vote.dm

```diff
@@ -75,15 +69,28 @@ SUBSYSTEM_DEF(vote)
 				for (var/non_voter_ckey in non_voters)
 					var/client/C = non_voters[non_voter_ckey]
 					if(C.prefs.preferred_map)
-						if(choices[C.prefs.preferred_map]) //No votes if the map isn't in the vote.
-							var/preferred_map = C.prefs.preferred_map
-							choices[preferred_map] += 1
-							greatest_votes = max(greatest_votes, choices[preferred_map])
-					else if(config.defaultmap)
-						if(choices[config.defaultmap]) //No votes if the map isn't in the vote.
-							var/default_map = config.defaultmap.map_name
-							choices[default_map] += 1
-							greatest_votes = max(greatest_votes, choices[default_map])
+						var/preferred_map = C.prefs.preferred_map
+						choices[preferred_map] += 1
+						greatest_votes = max(greatest_votes, choices[preferred_map])
+					else if(global.config.defaultmap)
+						var/default_map = global.config.defaultmap.map_name
+						choices[default_map] += 1
+						greatest_votes = max(greatest_votes, choices[default_map])
+			// Do you want to implement transfer? This calls the shuttle on crew vote. Great for long rounds.
+			// else if(mode == "transfer")
+			// 	var/factor = 1 // factor defines how non-voters are weighted towards calling the shuttle
+			// 	switch(world.time / (1 MINUTES))
+			// 		if(0 to 60)
+			// 			factor = 0.5
+			// 		if(61 to 120)
+			// 			factor = 0.8
+			// 		if(121 to 240)
+			// 			factor = 1
+			// 		if(241 to 300)
+			// 			factor = 1.2
+			// 		else
+			// 			factor = 1.4
+			// 	choices["Initiate Crew Transfer"] += round(non_voters.len * factor)
```

Comment:
Why is this commented out?

Reply:
I did want to include this but decided it should be atomized, oop. Removed